process/block/sovereign/outgoingOperations.go

```diff
@@ -148,16 +160,59 @@ func (op *outgoingOperations) isSubscribed(event data.EventHandler, txHash strin
 	return false
 }
 
-func createSCRData(topics [][]byte) []byte {
-	ret := topics[0]
-	for idx := 1; idx < len(topics[1:]); idx += 1 {
-		transfer := []byte("@")
-		transfer = append(transfer, topics[idx]...)
+func (op *outgoingOperations) getOperationData(event data.EventHandler) ([]byte, error) {
+	operation, err := op.newOperationData(event.GetTopics())
+	if err != nil {
+		return nil, err
+	}
 
-		ret = append(ret, transfer...)
+	evData, err := op.dataCodec.DeserializeEventData(event.GetData())
+	if err != nil {
+		return nil, err
 	}
 
-	return ret
+	operation.TransferData = evData.TransferData
+
+	operationBytes, err := op.dataCodec.SerializeOperation(*operation)
+	if err != nil {
+		return nil, err
+	}
+
+	return operationBytes, nil
+}
+
+func (op *outgoingOperations) newOperationData(topics [][]byte) (*sovereign.Operation, error) {
+	receiver := topics[1]
+
+	var tokens []sovereign.EsdtToken
```

Comment:
We should do:
1. Check the indexes are inside bonds so this code won't panic. If not enough topics, return error
2. Add consts values for 2/3